Case-3/Review/simulado-review.py

```diff
@@ -0,0 +1,105 @@
+'''
+1 - Média de acertos geral (em % e valor absoluto).
+2 - Média de acertos de cada cada aluno (em % e valor absoluto)
+3 - Total de acertos de cada aluno
+
+
+import pandas as pd
+openpyxl
+'''
+import sys
+import pandas as pd
+
+def corretor(path_respostas, path_gabarito):
+
+    # Carregando as planilhas e convertendo em DataFrames
+    df_respostas = pd.read_excel(path_respostas, engine ='openpyxl')
+    df_respostas = pd.DataFrame(df_respostas) 
+
+    df_gabarito = pd.read_excel(path_gabarito, engine ='openpyxl')
+    df_gabarito = pd.DataFrame(df_gabarito)
```

Comment:
Nessa parte eu iria extrair para uma funcao que faz a leitura das planilhas, porque nao sei se tu percebeu mas esse codigo ta fazendo praticamente a mesma coisa, criaria uma funcao mais ou menos assim.

```python

def ler_planilha(path_planilha):
    df = pd.read_excel(path_planilha, engine ='openpyxl')
    return pd.DataFrame(df)  

```
Entao o codigo ficaria mais ou menos assim:

```suggestion

    df_respostas = ler_planilha(path_respostas)
    df_gabarito = ler_planilha(path_gabarito)
    
```